src/openzaak/components/zaken/models/zaken.py

```diff
@@ -897,6 +909,17 @@ def save(self, *args, **kwargs):
 
         super().save(*args, **kwargs)
 
+    def clean(self):
```

Comment:
Could you also add this for the other models where this is used if it applies? E.g. I see that the validator is also used for ZaakEigenschap, Resultaat and ZaakObject